run_browser_with_docker.sh

```diff
@@ -17,7 +17,7 @@
 # Initializes port_mappings with all running application ports using the Service Manager v2 status command.
 # Appends ZAP_PORT 11000 to ./run-zap-spec.sh
 #######################################
-port_mappings=$(sm2 -s --format-plain | grep -E 'PASS|BOOT' | awk '{ print $3"->"$3 }' | paste -sd "," -)
+port_mappings=$(sm2 --status | grep PASS | awk '{ print $8"->"$8 }' | paste -sd "," -)
```

Comment:
This command doesn't seem to grab the ports any more. You can test it by pasting it into terminal; it should give you the open ports of SM services with an arrow in between, separated by commas, e.g. `1234->1234,5678->5678`

Seems like the previous command was fine?